qutebrowser/completion/completer.py

```diff
@@ -197,13 +197,19 @@ def schedule_completion_update(self):
         For performance reasons we don't want to block here, instead we do this
         in the background.
         """
-        if (self._cmd.cursorPosition() == self._last_cursor_pos and
+        cmd = self._cmd.text().split(' ', 1)
```

Comment:
You could probably do  `cmd, _sep, rest = self._cmd.text().partition(' ')` instead, then the `if` below will become a bit simpler.